### PR TITLE
Fix indentLine for latest monaco version

### PR DESCRIPTION
**Pull request by CGNonofr**

### CODE REVIEW

**Review by brijeshb42 — COMMENTED** [review]
Will this fix work for older versions of monaco?

**Comment by CGNonofr** [comment]
Nop,  `_getViewModel` is only defined starting from monaco-editor@0.21, and `_getCursors` is not more defined starting from monaco-editor@0.21

What should we do about it? is it an issue?

**Comment by brijeshb42** [comment]
Supporting new version this way will break for majority of users who are still in the old version. Checking for `_getViewModel` first and then using it will be better.

**Comment by CGNonofr** [comment]
Is it what you expected?

**Comment by CGNonofr** [comment]
Do you want me to squash the commits?

File: src/cm_adapter.js [hunk 1]
```diff
@@ -1104,13 +1104,20 @@ class CMAdapter {
 
   indentLine(line, indentRight = true) {
     const { editor } = this
-    const cursors = editor._getCursors();
+    let cursorConfig;
+    if (editor._getViewModel != null) {
```
Comment (brijeshb42) [inline src/cm_adapter.js:1108]: Remove the null check. Just checking for `editor._getViewModel` would be fine as it would be `undefined` otherwise.

**Comment by brijeshb42** [comment]
> Do you want me to squash the commits?

Yes please.

**Comment by CGNonofr** [comment]
Changed and rebased

**Pull request merged**